es-ds-docs/nuxt.config.ts

```diff
@@ -8,5 +8,5 @@ export default defineNuxtConfig({
     // https://nuxt.com/docs/getting-started/deployment#static-hosting
     ssr: true,
 
-    modules: ['./modules/auto-import-eslint.ts'],
+    modules: ['./modules/auto-import-eslint.ts', '@nuxt/image'],
```

Comment:
Could `@nuxt/image` live in the `es-ds-components` layer?

Reply:
Maybe, but it is possible to use design system components in a micro frontend without needing nuxt image, so maybe we'd better leave that up to downstream repos. I'd guess _most_ downstream repos would end up using nuxt image, but maybe not all.